package.json

```diff
@@ -106,7 +107,10 @@
     "yauzl": "^2.10.0"
   },
   "engines": {
-    "node": "8.* || >= 10.*"
```

Comment:
Is this actually going to require 12+?

Reply:
@rwwagner90 As far as consumers go, it doesn't affect them, since they only consume the built artifacts. The Volta config is pinned to Node 12 so if you have it installed (you should, it's great 😄), it will automatically pick it up. That said, nothing in the deps actually require Node 12 so I'll change it back to `>= 10.*`. However, CI is running Node 12 (via Volta) so it's an untested setup.